src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -105,6 +105,21 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return ' '.join(msg_chunks)
 
 
+def _nodes_from_rst(state: statemachine.State, rst: str) -> List[nodes.Node]:
```

Comment:
Extra feedback that I missed in the original PR. How about renaming `rst` into `rst_source` so that it's more explicit?

Reply:
@ewjoachim WDYT?

Reply:
yeah totally :)

Reply:
```suggestion
def _nodes_from_rst(state: statemachine.State, rst_source: str) -> List[nodes.Node]:
```